src/app/gateways/components/RuleMatch.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <dl class="match-definition-list">
+    <template v-if="props.match.method">
+      <div class="match">
+        <dt>
+          <KBadge appearance="neutral">
+            Method
+          </KBadge>
+        </dt>
+
+        <dd>
+          <KBadge>{{ props.match.method }}</KBadge>
+        </dd>
+      </div>
+    </template>
+
+    <template v-if="props.match.path">
+      <div class="match">
+        <dt>
+          <KBadge appearance="neutral">
+            Path
+          </KBadge>
+        </dt>
+
+        <dd>
+          <span class="key">Type:</span> <span>{{ props.match.path.type }}</span>
+          <span class="key">Value:</span> <code>{{ props.match.path.value }}</code>
+        </dd>
+      </div>
+    </template>
+
+    <template v-if="props.match.queryParams && props.match.queryParams.length > 0">
+      <div
+        v-for="(param, index) in props.match.queryParams"
+        :key="index"
+      >
```

Comment:
I know a `div` is fine within a `dl` in order to wrap the dt/dd combo. Not totally sure about nested divs any idea?

Reply:
Ah, good point. I don’t know for sure, but in this case, we don’t need two levels anyway, so I removed one.